src/ast/ddl.rs

```diff
@@ -601,6 +625,14 @@ pub enum ColumnOption {
         generation_expr: Option<Expr>,
         generation_expr_mode: Option<GeneratedExpressionMode>,
     },
+    /// BigQuery specific: Explicit column options in a view [1] or table [2]
+    /// Syntax
+    /// ```sql
+    /// OPTIONS(description="field desc")
+    /// ```
+    /// [1]: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#view_column_option_list
+    /// [2]: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#column_option_list
+    SqlOptions(Vec<SqlOption>),
```

Comment:
Given the SQL is `OPTION`s what would you think about keeping the AST similar?

```suggestion
    Options(Vec<SqlOption>),
```

Reply:
Done!